packages/core/useQuickSort/index.ts

```diff
@@ -0,0 +1,27 @@
+import type { UseSortCompareFn } from '../useSort'
+import { useSortWrapFn } from '../useSort'
+
+export function quickSort<T>(source: T[], compareFn: UseSortCompareFn<T>): T[] {
```

Comment:
I am a bit hesitate about this. Could we force on the general `useSorted` first and then think about the possibly way to support more alrigthom?

Reply:
Also, I feel JS's native sort should be quite fast already (https://stackoverflow.com/a/37245185), I have never written a sorting algorithm in JS actually. Is there any benefit to implementing a quick sort on our own?

Reply:
> I am a bit hesitated about this. Could we force on the general `useSorted` first and then think about the possible way to support more algorithms?

ok~.

> Also, I feel JS's native sort should be quite fast already (https://stackoverflow.com/a/37245185), I have never written a sorting algorithm in JS actually. Is there any benefit to implementing a quick sort on our own?

Not sure. Every browser type has it's own javascript engine implementation.
Just wanted to provide a `useSorted` implementation here.